source/user_guide.rst

```diff
@@ -20,115 +17,95 @@ Who should read this
 Getting Started
 ---------------
 
-To start, please follow dojot's installation guide in
-:doc:`install/compose_guide`. There you should find how to properly download a
-working copy of the components, how to minimally configure them, how to start
-them up and how to check whether they are working.
+To start, please follow dojot's installation guide in :doc:`install/compose_guide`. There you should find how to
+properly download a working copy of the components, how to minimally configure them, how to start them up and how to
+check whether they are working.
 
 dojot basics
 ------------
 
-Before using dojot, you should be familiar with some basic operations and
-concepts. They are very simple to understand and use, but without them, all
-operations might become obscure and senseless. It is advisable to checkout our
+Before using dojot, you should be familiar with some basic operations and concepts. They are very simple to understand
+and use, but without them, all operations might become obscure and senseless. It is advisable to checkout our
```

Comment:
Just as a random note - if users need to understand our internal architecture to use the platform, we're not doing a good enough job of abstracting complexity (the "easy" part might be missing).

Reply:
Ok, I'll rewrite this part. It seems that the internal components are the 'basic' stuff.